scripts/utilities/getPackages/index.test.js

```diff
@@ -6,18 +6,13 @@ describe(`Publish Script - getPackages`, () => {
 
   it("should return array of all package's paths", () => {
     jest.mock('shelljs', () => ({
-      exec: () =>
-        JSON.stringify({
-          '@bbc/psammead-consent-banner': {
-            location: 'packages/components/psammead-consent-banner',
-          },
-          '@bbc/psammead-brand': {
-            location: 'packages/components/psammead-brand',
-          },
-          '@bbc/psammead-bulleted-list': {
-            location: 'packages/components/psammead-bulleted-list',
-          },
-        }),
+      exec: () => ({
+        stdout: [
+          '{"location":"packages/components/psammead-consent-banner","name":"@bbc/psammead-consent-banner"}',
+          '{"location":"packages/components/psammead-brand","name":"@bbc/psammead-brand"}',
+          '{"location":"packages/components/psammead-bulleted-list","name":"@bbc/psammead-bulleted-list"}',
```

Comment:
Look to reimplementing the json stringify

Reply:
I don't think we can use `JSON.stringify()` for these, as the output that we are mocking is provided in an [NDJSON](http://ndjson.org/) format, not the standard JSON format, hence the use of the `.join('\n')` to mock the behaviour of the [`yarn workspaces list --json`](https://yarnpkg.com/cli/workspaces/list) command.

@jroebu14 @andrewscfc